### ISSUE TITLE
modeldata is empty in Storage account even after multiple webservice endpoint calls.

### ISSUE THREAD

**Issue by rgknp** [microsoft/MCW-ML-Ops#35]
Exercise 9: Examining deployed model performance
Task 3: Check the data collected

I followed the steps to setup Application Insights and went to Storage Account in the resource group where all the lab resources are placed. Even after multiple successful calls to deployed Model endpoint, there are no results saved in the storage account. modeldata container is empty.

![Screenshot 2020-10-01 221656](https://user-images.githubusercontent.com/5192916/94881877-84ef0700-0434-11eb-8397-3905b929f968.jpg)

**Comment by shirolkar** [comment]
@rgknp I am not able to reproduce this issue. Can you confirm:

Task 1: Activate App Insights and data collection on the deployed model 
Confirm that the webservice_url and webservice_key you provided in the notebook is for AKS service and not ACI service.
Confirm that task 1 worked without an errors in the last cell of the notebook.

Task 2: Check Application Insights telemetry
Confirm that you are getting back results as shown in step 7 for the AKS service.

Also the calls you are making to is to the AKS service and not ACI service.

Thanks.

**Comment by rgknp** [comment]
I tried again and it works. Somehow even after multiple refresh I wasn't able to see telemetry. Today when I tried it created a folder within modeldata.

On a different note, it appears like model always returns prediction "0.57746685" no matter what input is passed. Even with the inputs.csv currently present I see there are two input types but prediction is always the same.

**Comment by shirolkar** [comment]
@rgknp Are you referring to making predictions using the trained model in Deep Learning with Text notebook? Or the deployed models from the release pipelines as shown in Test Deployment notebook? Note that you should train the models with lr=0.001 and epochs = 5 to get good accuracy. The training data is limited and very simple, so we should get close to 100% accuracy. 

For making inferences the input to the deployed service needs to be a list of descriptions / sentences, for example:
['poor condition carbon fiber component manufactured in 2014',
 'steel component manufactured in 2009 in new condition',
 'manufactured in 2015 made of iron in good condition']

If you are using model trained in the Deep Learning with Text notebook then the input to the model needs to be tokenized and padded sequences of shape: ['None', 100].

**Comment by DawnmarieDesJardins** [comment]
Closing issue as resolved with merge of PR #36
